config/samples/kustomization.yaml

```diff
@@ -1,3 +1,4 @@
 ## This file is auto-generated, do not modify ##
 resources:
+- monitoring_v1alpha1_clusterurlmonitor.yaml
```

Comment:
Why was that actually not present before?

Reply:
Nope, Manuel added it and it was not in the kustomization file (I guess he assumed it takes everything in the folder)